cmd/keytransparency-sequencer/server.go

```diff
@@ -16,6 +16,7 @@ package main
 
 import (
 	"flag"
+	"log"
```

Comment:
Should be glog, which is already imported?

Reply:
done